doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -52,6 +52,7 @@ location / {
     security:
         enable_authenticator_manager: true
     ```
+  `PreAuthenticatedAdminSessionListener` has been deprecated and will be removed in Pimcore 11.
```

Comment:
```suggestion
  `[Authentication] PreAuthenticatedAdminSessionListener` has been deprecated and will be removed in Pimcore 11.
```

Reply:
> Does this service also need a deprecation message?

I would avoid that since this used internally and then it triggers the deprecations on compilation.